### ISSUE TITLE
Scheduling the LR for PPCA models

### ISSUE THREAD

**Issue by ordabayevy** [cellarium-ai/cellarium-ml#9]
(unrelated to this PR -- but I have a feeling that scheduling the LR e.g. w/ the usual cosine annealing might do wonders with PPCA. why? because it's a simple model w/ a convex loss and after the initial convergence to a rough solution, annealing can let the "ball" roll down to the global minimum. It'd be great to try this out in your experiments!).

_Originally posted by @mbabadi in https://github.com/cellarium-ai/scvi-distributed/pull/6#discussion_r1099041575_

**Comment by ordabayevy** [comment]
Some useful links about lr schedulers:
- https://www.kaggle.com/code/residentmario/cosine-annealed-warm-restart-learning-schedulers
- https://www.kaggle.com/code/residentmario/one-cycle-learning-rate-schedulers/notebook
- https://sgugger.github.io/the-1cycle-policy.html
- https://timm.fast.ai/SGDR
- [SGDR: STOCHASTIC GRADIENT DESCENT WITH WARM RESTARTS](https://arxiv.org/pdf/1608.03983v5.pdf)
- [A DISCIPLINED APPROACH TO NEURAL NETWORK HYPER-PARAMETERS](https://arxiv.org/pdf/1803.09820.pdf)
- [Super-Convergence: Very Fast Training of Neural Networks Using Large Learning Rates](https://arxiv.org/pdf/1708.07120.pdf)
